app/src/main/res/drawable/ic_copy.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="?attr/colorControlNormal"
```

Comment:
I like to also format these kind of files for consistency and readability.

Reply:
no idea why it didn't format the file FeelsDankMan